azure/aks-kubernetes.md

```diff
@@ -9,7 +9,7 @@ This can be done by AKS, but it is recommended to use the integration with Azure
 Using AD lets you manage users in a central place that can be used across many different Azure services.
 
 1.  https://kubernetes.io/docs/reference/access-authn-authz/controlling-access/
-2.  https://docs.microsoft.com/en-us/azure/aks/concepts-identity
+2.  https://docs.microsoft.com/en-gb/azure/aks/security-baseline#31-maintain-an-inventory-of-administrative-accounts
```

Comment:
I think the `en-us` docs are probably the canonical ones, given that you're forwarded to them if you strip the language tag.

Reply:
Sure, i feel like the is a little more relevant, but now the table is clearer i am not too fussed